doc/architecture/consensus/index.rst

```diff
@@ -20,6 +20,18 @@ CFT parameters can be configured when starting up a network (see :doc:`here </op
 - ``consensus.message_timeout`` is the Raft heartbeat timeout. The Raft leader sends heartbeats to its followers at regular intervals defined by this timeout. This should be set to a significantly lower value than ``consensus.election_timeout``.
 - ``consensus.election_timeout`` is the Raft election timeout. If a follower does not receive any heartbeat from the leader after this timeout, the follower triggers a new election.
 
+Extensions for Omission Faults
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
A link to a definition of omission fault would be a good thing to have in this paragraph.